packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -164,6 +166,11 @@ class Draggable<T> extends StatefulWidget {
   /// Where this widget should be anchored during a drag.
   final DragAnchor dragAnchor;
 
+  /// Whether to ignore the semantics of the [feedback] widget.
+  ///
+  /// Defaults to true.
+  final bool ignoringFeedbackSemantics;
```

Comment:
It's not clear to me what this actually does or when you want to set it to false. Might need more documentation?

Reply:
Updated comment